corehq/ex-submodules/pillowtop/es_utils.py

```diff
@@ -45,28 +45,29 @@ def to_json(self):
         return json
 
 
-def set_index_reindex_settings(es, index):
+def set_index_reindex_settings(index):
     """
     Set a more optimized setting setup for fast reindexing
     """
-    return ElasticsearchInterface(es).update_index_settings_reindex(index)
+    return manager.index_configure_for_reindex(index)
 
 
-def set_index_normal_settings(es, index):
+def set_index_normal_settings(index):
     """
     Normal indexing configuration
     """
-    return ElasticsearchInterface(es).update_index_settings_standard(index)
+    return manager.index_configure_for_standard_ops(index)
 
 
 def initialize_index_and_mapping(es, index_info):
-    index_exists = es.indices.exists(index_info.index)
+    # TODO: Remove es param as it is unused
```

Comment:
will this be done separately?

Reply:
Yep. Tests are a heavy consumer of this function. I have thought of a followup PR in which more cleanup would be done which will resolve this TODO.